pynsee/utils/config.py

```diff
@@ -3,11 +3,13 @@
 import json
 import logging
 import os
-import requests
+#import requests
 import urllib3
 
 from typing import Any, Optional, Union
 
+from pynsee.utils._request_insee import _request_insee
```

Comment:
You cannot do that, _request_insee depends on config files.
You could import it within the function if it is really necessary, but I'm not sure why it would be, could you explain?